cos_alerter/alerter.py

```diff
@@ -120,9 +120,10 @@ def initialize():
         #     ...
         # }
         state["clients"] = {}
-        for client in config["watch"]["clients"]:
+        for client_data in config["watch"]["clients"]:
+            client_id = client_data["id"]
```

Comment:
I'm not clear on the details but have you considered loading the config with a pydantic model?
You'd get static fields and some other benefits.

Reply:
Discussed this with the team and agreed to have a separate issue for it to load the config with a pydantic model